engine/src/main.cpp

```diff
@@ -323,7 +323,7 @@ int main( int argc, char *argv[] )
 
     // when the program name is `vegastrike-engine` then enforce that the data directory must be specified
     // if the program name is `vegastrike` then enable legacy mode where the current path is assumed.
-    legacy_data_dir_mode = (program_name == "vegastrike") || (program_name == "vegastrike.exe");
+    legacy_data_dir_mode = true; //(program_name == "vegastrike") || (program_name == "vegastrike.exe");
```

Comment:
Um, what? Why are we commenting out this logic? This was put here intentionally, by @BenjamenMeyer .

Reply:
yes, that was a means of detecting legacy mode to fall back for old behavior with running it as the main binary is now `vegastrike-engine`. CMake takes care of creating the `vegastrike` binary as a copy during one of its steps (IIRC in the install process).

Reply:
You are both absolutely right about this line. It's a hack I put because I was too lazy to figure out why the game wouldn't run and I was a good boy not to commit it until I missed it in the merge.